sllurp/llrp.py

```diff
@@ -330,6 +330,8 @@ def __init__(self, config, transport_tx_write=None,
 
         self.tx_power_table = []
 
+        self.llrp_keepalive_interval_ms = config.llrp_keepalive_interval_ms
```

Comment:
This a client configuration value, so I don't think that there is value to have the variable value duplicated here in LLRPClient.